src/numbuf/python/src/pynumbuf/numbuf.cc

```diff
@@ -128,32 +127,26 @@ static PyObject* write_to_buffer(PyObject* self, PyObject* args) {
   if (!PyMemoryView_Check(memoryview)) { return NULL; }
   Py_buffer* buffer = PyMemoryView_GET_BUFFER(memoryview);
   auto target = std::make_shared<FixedBufferStream>(
-      reinterpret_cast<uint8_t*>(buffer->buf), buffer->len);
-  int64_t body_end_offset;
-  int64_t header_end_offset;
-  ARROW_CHECK_OK(ipc::WriteRecordBatch((*batch)->columns(), (*batch)->num_rows(),
-      target.get(), &body_end_offset, &header_end_offset));
-  return PyLong_FromLong(header_end_offset);
+      reinterpret_cast<uint8_t*>(buffer->buf) + sizeof(int64_t),
```

Comment:
`sizeof(int64_t)` appears all over, should it be stored in a more descriptive variable name? This is worth documenting.